addons/web/static/src/core/utils/xml.js

```diff
@@ -1,5 +1,21 @@
 /** @odoo-module **/
 
+// Both variables are used by the hasParsingError function.
+const errorneousParse = new DOMParser().parseFromString("<", "text/xml");
+const parsererrorNS = errorneousParse.getElementsByTagName("parsererror")[0].namespaceURI;
```

Comment:
erroneous

Reply:
this line triggers an error on firefox.

Note that "a valid xml with a <parsererror> tag" isn't a usecase we need to handle, so I'd go for `invalidXml.getElementsByTagName("parsererror")[0]`